tests/unit/test_habit.py

```diff
@@ -0,0 +1,146 @@
+from fastapi.testclient import TestClient
```

Comment:
sold could add a test for 404, seems those are the only lines missing from coverage `$ pytest --cov=habits --cov-report term-missing` (needs `pip install pytest-cov`):

```
---------- coverage: platform darwin, python 3.9.5-final-0 -----------
Name                 Stmts   Miss  Cover   Missing
--------------------------------------------------
habits/__init__.py       0      0   100%
habits/db.py             7      1    86%   11
habits/main.py          77      8    90%   22, 26-27, 54, 62, 95, 109, 117
habits/models.py        22      0   100%
habits/payloads.py       1      0   100%
--------------------------------------------------
TOTAL                  107      9    92%


================================================================================= 16 passed in 0.57s ==
```